cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -0,0 +1,156 @@
+    }
+
+    @After(order = 1, value = "@BQ_SOURCE_TEST")
+    public static void deleteTempSourceBQTable() throws IOException, InterruptedException {
+        String bqSourceTable = PluginPropertyUtils.pluginProp("bqSourceTable");
+        BigQueryClient.dropBqQuery(bqSourceTable);
+        BeforeActions.scenario.write("BQ source Table " + bqSourceTable + " deleted successfully");
+        PluginPropertyUtils.removePluginProp("bqSourceTable");
+    }
+    private static void createSourceBQTableWithQueries(String bqCreateTableQueryFile, String bqInsertDataQueryFile)
+            throws IOException, InterruptedException,NullPointerException {
+        String bqSourceTable = "E2E_SOURCE_" + UUID.randomUUID().toString().substring(0,5).replaceAll("-", "_");
+
+        String createTableQuery = StringUtils.EMPTY;
+        try {
+            createTableQuery = new String(Files.readAllBytes(Paths.get(TestSetupHooks.class.getResource
+                    ("/" + bqCreateTableQueryFile).toURI()))
+                    , StandardCharsets.UTF_8);
+            createTableQuery = createTableQuery.replace("DATASET", PluginPropertyUtils.pluginProp("dataset"))
+                    .replace("TABLE_NAME", bqSourceTable);
+        } catch (Exception e) {
+            e.printStackTrace();
+            BeforeActions.scenario.write("Exception in reading " + bqCreateTableQueryFile + " - " + e.getMessage());
+            Assert.fail("Exception in BigQuery testdata prerequisite setup " +
+                    "- error in reading create table query file " + e.getMessage());
+        }
+
+        String insertDataQuery = StringUtils.EMPTY;
+        try {
+            insertDataQuery = new String(Files.readAllBytes(Paths.get(TestSetupHooks.class.getResource
+                    ("/" + bqInsertDataQueryFile).toURI()))
+                    , StandardCharsets.UTF_8);
+            insertDataQuery = insertDataQuery.replace("DATASET", PluginPropertyUtils.pluginProp("dataset"))
+                    .replace("TABLE_NAME", bqSourceTable);
+        } catch (Exception e) {
+            BeforeActions.scenario.write("Exception in reading " + bqInsertDataQueryFile + " - " + e.getMessage());
+            Assert.fail("Exception in BigQuery testdata prerequisite setup " +
+                    "- error in reading insert data query file " + e.getMessage());
+
+        }
+        BigQueryClient.getSoleQueryResult(createTableQuery);
+        try {
+            BigQueryClient.getSoleQueryResult(insertDataQuery);
+        } catch (NoSuchElementException e) {
+            // Insert query does not return any record.
+            // Iterator on TableResult values in getSoleQueryResult method throws NoSuchElementException
+        }
+        System.out.println(bqSourceTable);
+        PluginPropertyUtils.addPluginProp("bqSourceTable", bqSourceTable);
+//        BeforeActions.scenario.write("BQ Source Table " + bqSourceTable + " created successfully");
+    }
+
```

Comment:
Remove the space.

Reply:
done
commit link- b03766131af7a687ad66f495a6eac551f74c15d8